spec/shr_genetics.txt

```diff
@@ -15,12 +15,13 @@ CodeSystem:     UCUM = http://unitsofmeasure.org
  */
 EntryElement:	GeneticMarkerAnalysisResult
 Based on:		LaboratoryObservation
-Concept:		MTH#C0017393  // should be a finding (Q+A), not an observable
-Description:	"A finding with respect to variation in a segment of DNA with an identifiable physical location on a chromosome."
-				FindingTopicCode is MTH#C0017393 "A segment of DNA with an identifiable physical location on a chromosome and whose inheritance can be followed. (source: NCI Metathesaurus)"
+Concept:		MTH#C0679560 "Genetic Testing"
+Description:	"The process of isolating and testing the DNA of an embryo or a person to detect the presence of genetic alterations or defects that may predispose to the future development of a disease or disorder. (source: NCI Thesaurus)"
+				FindingTopicCode from https://www.ncbi.nlm.nih.gov/gtr if covered // the identifier from the Genetic Test Registry
+				FindingMethod from GeneticTestMethodVS if covered
 0..0			FindingResult // panel, no numerical or coded result
 0..0			SpecificFocusOfFinding
-//  What gene are we looking at?
+//  MK: What gene are we looking at? mlt: identified in two places: 1) FindingTopicCode through the GTR ID, 2) in the GeneName element below.
 0..1			GenomicSourceClass
```

Comment:
I'm not entirely comfortable that the FindingTopicCode is the gene name. The gene is certainly the target of inquiry, but it like saying the topic of a blood pressure measurement is "blood" or the topic of a cognitive function test is the brain. The FindingTopicCode is supposed to be the code for the test.